client/src/indexer/grpc.rs

```diff
@@ -0,0 +1,14 @@
+use notify::DebouncedEvent;
+
+tonic::include_proto!("server");
+
+impl From<&DebouncedEvent> for FileEventType {
+    fn from(e: &DebouncedEvent) -> Self {
+        match e {
+            DebouncedEvent::Create(_) => FileEventType::Create,
+            DebouncedEvent::Write(_) => FileEventType::Update,
+            DebouncedEvent::Remove(_) => FileEventType::Delete,
+            _ => FileEventType::Unknown,
```

Comment:
on prend en compte le rename/déplacement de fichier ou pas ?

Reply:
Yes for sure, but at the moment the protobuf definitions doesn't suppport it. When this will be implemented, we have to add the missing case in the pattern matching here.

Reply:
ok no problem, we just need to keep in mind so. ok to me